lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -5,6 +5,11 @@ module Helper
     # Helper methods to execute git-related operations
     #
     module GitHelper
+      # Fallback default branch of the client repository. It's currently set to 'develop' for
+      # backwards compatibility.
+      # TODO: Set to 'trunk' for the next major release.
```

Comment:
👍 

Side note about something I've been thinking about for a while now that we use SemVer is how to keep track of those cases of "let's deprecate X first and only remove it on next major" or, like here, "let's wait for next major to change this". We have had a few of those cases recently where we wrote some `TODO:` or `FIXME:` sometimes (not sure if every time…)… but it will be quite easy to forget to tackle every one of those next time we do a major version.
So if anyone has any suggestion on how to track those better, I'm all ears…
 - A long-lived GitHub issue for each next major we plan to do, and we'd regularly edit the issue description to add more and more checkboxes with links to lines/methods to change before we do a new release?
 - A note at the very top of the release notes, saying "this and that will have to be deleted on next major release"? (not as nice, but at least it'd be hard to miss when releasing versions)
 - A dedicated `.md` file in the repo – which could double-down as a "Migration Documentation" to tell people what they need to do or be prepared to for each major version
 - A convention on how to name our `TODO:`/`FIXME:` to annotated those, e.g. `FIXME: [NextMajor] blahblah"?

Reply:
I like the last option. If we use `TODO` sparingly (I have been using `TODO` because it's syntax highlighted for me, but maybe it works with `FIXME` as well) we could look at all `TODO`s before each major release cut and either address them (in cases like this) or move them into issues and clear them up before each release.

Dedicated `.md` is also 👍 from me.

Reply:
I wonder if we could have a section in the release notes for this? That way we don't need to remember to check another file when cutting a breaking build?